service/history/api/queryworkflow/api.go

```diff
@@ -241,6 +248,27 @@ func Invoke(
 	}
 }
 
+func queryWillTimeoutsBeforeTaskStart(
```

Comment:
Let's be more specific about task:
```suggestion
func queryWillTimeoutsBeforeFirstWorkflowTaskStart(
```

Reply:
I think it is too verbose, but ok